src/components/Dashboard/DashboardHeader.jsx

```diff
@@ -69,6 +70,13 @@ const DashboardHeader = ({
   actions,
   onDashboardAction,
 }) => {
+  if (__DEV__) {
+    warning(
+      false,
+      'Dashboard component has been deprecated and will be removed in the next release of `carbon-addons-iot-react`.'
+    );
+  }
+
```

Comment:
Lets not deprecate this one as it's a reusable piece we'll want to export/make available and make future improvements to.

Reply:
Removed the deprecation. Thanks!